docs/memory-safety-continuum/memory-safety-continuum-draft2.md

```diff
@@ -101,14 +106,30 @@ The amount software that has already been produced is staggering - and it is onl
 * Using a mutation tester such as [cargo-mutants](https://github.com/sourcefrog/cargo-mutants)
 * Using [CodeQL](https://codeql.github.com/) for the [languages that CodeQL supports](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/)
 * Using [DevSkim](https://github.com/microsoft/devskim) IDE extensions/language analyzers
+* [More best practices](https://github.com/ossf/Memory-Safety/blob/main/docs/best-practice-memory-safe-by-default-languages.md)
 
 #### Memory safe by default language automated tooling to provide additional checks to your dependencies
 
 * Using a fuzzer such as [AFL++](https://github.com/AFLplusplus/AFLplusplus) on both your own code and third party code
 
 ### 2. Using Memory Safe by Default Languages to interface with Non-Memory Safe By Default Languages
 
-TO DO
+#### General interfacing best practices
+
+* Separating all FFIs into a separate crate/module/package so that it can be audited in isolation. All exported symbols from that crate/module/package should be memory safe. [Source](https://github.com/ossf/Memory-Safety/issues/36#issuecomment-2477083785)
```

Comment:
should the continuum document be referencing an issue comment? feels a bit too "in the details", wdyt?